scripts/dcp1-version/populate_dcp_version.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import os
+import uuid
+from datetime import datetime
+
+from bson import JAVA_LEGACY, CodecOptions
+from pymongo import MongoClient
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+DCP1_GS_FILES_LS = os.environ.get('DCP1_GS_FILES_LS')
```

Comment:
W@hat does the `LS` suffix mean?

Reply:
I renamed ls to list. I was trying to shorten it.